cvat-ui/src/components/annotation-page/standard-workspace/objects-side-bar/object-item.tsx

```diff
@@ -433,7 +433,7 @@ function ItemButtonsComponent(props: ItemButtonsComponentProps): JSX.Element {
                     <Col>
                         <Tooltip title={`Switch lock property ${switchLockShortcut}`}>
                             { locked
-                                ? <Icon type='lock' onClick={unlock} theme='filled'/>
+                                ? <Icon type='lock' onClick={unlock} theme='filled' />
```

Comment:
Can we do the same for occluded or antd doesn't have ``filled`` theme for this icon?

Reply:
@bsekachev, there is no such icon